rest/api_test.go

```diff
@@ -8034,57 +8034,8 @@ func TestBasicAttachmentRemoval(t *testing.T) {
 		assert.Equal(t, attBodyExpected, attBodyActual)
 	}
 
-	rawDocWithAttachmentAndSyncMeta := func() []byte {
-		return []byte(`{
-   "_sync": {
-      "rev": "1-5fc93bd36377008f96fdae2719c174ed",
-      "sequence": 2,
-      "recent_sequences": [
-         2
-      ],
-      "history": {
-         "revs": [
-            "1-5fc93bd36377008f96fdae2719c174ed"
-         ],
-         "parents": [
-            -1
-         ],
-         "channels": [
-            null
-         ]
-      },
-      "cas": "",
-      "attachments": {
-         "hi.txt": {
-            "revpos": 1,
-            "content_type": "text/plain",
-            "length": 2,
-            "stub": true,
-            "digest": "sha1-witfkXg0JglCjW9RssWvTAveakI="
-         }
-      },
-      "time_saved": "2021-09-01T17:33:03.054227821Z"
-   },
-  "key": "value"
-}`)
-	}
-
 	createDocWithLegacyAttachment := func(docID string, rawDoc []byte, attKey string, attBody []byte) {
```

Comment:
Nitpick: Can the calls to this inline function be removed and calls to the inline one be replaced with the new one (by passing the testing and rt variables)?

Reply:
Yeah, when this gets ported to master I think that would be reasonable.  Omitted for this version to reduce the changes made on 3.0.3.